dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java

```diff
@@ -1498,6 +1498,39 @@ public void testRefreshingUserMetadata() {
         Assert.assertEquals("One user should have this info now, there are " + count, 1, count);
     }
 
+    @Test
+    public void launchToolChecksumValidation() {
+        // manual publish the tool
+        Client.main(new String[] { "--config", ResourceHelpers.resourceFilePath("config_file.txt"), "tool", "publish", "--entry",
+            "quay.io/dockstoretestuser/test_input_json", "--script" });
+
+        // refresh the tool
+        Client.main(new String[] { "--config", ResourceHelpers.resourceFilePath("config_file.txt"), "tool", "refresh", "--entry",
+            "quay.io/dockstoretestuser/test_input_json", "--script" });
+
+        // launch the tool
+        systemOutRule.clearLog();
+        Client.main(new String[] { "--config", ResourceHelpers.resourceFilePath("config_file.txt"), "tool", "launch", "--entry",
+            "quay.io/dockstoretestuser/test_input_json", "--json", ResourceHelpers.resourceFilePath("tool_hello_world.json"), "--script" });
+        assertTrue("Output should indicate that checksums have been validated",
+            systemOutRule.getLog().contains("Validated checksums") && !systemOutRule.getLog().contains("Cannot validate the checksum of the locally downloaded descriptor."));
```

Comment:
Use constant string shared with implementation so that future developers can tell that this test is associated with the implementation below and to avoid instant breakage

Reply:
[fixed](https://github.com/dockstore/dockstore-cli/blob/7f6c85a4a186819af1abeaf2f795939c1fb693b4/dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java#L1505). Added final strings at start of test.